api/pom.xml

```diff
@@ -165,6 +165,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.pdfbox</groupId>
+            <artifactId>pdfbox</artifactId>
+            <version>2.0.1</version>
```

Comment:
We need the version 2.0.24 or 2.0.25 as the previous versions have some High Risk vulnerabilities reported.

Reply:
Sure, i will change that right now and push it